sample-app/README.md

```diff
@@ -9,15 +9,27 @@ You can start the application as you for any other regular Spring Boot applicati
 1. From the command line, under the sample application project, run `mvn spring-boot:run`
 1. From your favorite IDE, run class `com.test.Application`
 
-## Teting it
+## Testing it
 
 Send a **GET** request to the endpoint below.
 
 [http://localhost:8080/sample-app/echo/echo](http://localhost:8080/sample-app/echo/echo)
 
 You should receive a response message with this payload as result:
 
-    {
-        "wordId": 1,
-        "wordString": "Lua"
-    }
+``` json
+{
+    "wordId": 1,
+    "wordString": "Lua"
+}
+```
+   
+## Automated tests
+
+ 1. Unit tests are located in `src/test/java` and end with `Test`. Execute the tests with `mvn test`
+ 
```

Comment:
Please, remove this blank line and rename item 2 to 1. GitHub markdown numbers them accordingly, example:

1. Item 1
1. Item 2